app/assets/v2/js/pages/bounty_details2.js

```diff
@@ -169,28 +169,32 @@ Vue.mixin({
         });
       }
     },
-    fulfillmentComplete: function(fulfillment_id, amount) {
-
+    fulfillmentComplete: function(fulfillment_id, amount, event) {
       let vm = this;
-
       const token_name = vm.bounty.token_name;
       const decimals = tokenNameToDetails('mainnet', token_name).decimals;
-
       const payload = {
         amount: amount * 10 ** decimals,
         token_name: token_name,
         bounty_owner_address: vm.bounty.bounty_owner_address
       };
-
       const apiUrlBounty = `/api/v1/bounty/payout/${fulfillment_id}`;
 
+      event.target.disabled = true;
```

Comment:
what's the event.target for ? to disable the button ?

Reply:
yes